index.html

```diff
@@ -1,33 +1,35 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<title></title>
+	<title>lass</title>
 	<script src="https://kit.fontawesome.com/b8019b4db4.js" crossorigin="anonymous"></script>
 	<link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
 	<header class="container">
-		<div id="logo">My Logo</div>
+		<div id="logo"><a href="#">My Logo</a></div>
 		<nav>
-			<button><i class="fas fa-bars" id="bar"></i></button>
+			<button type="button"><i class="fas fa-bars" id="bar"></i></button>
```

Comment:
- [ ] OPTIONAL: In this project, you will have to strictly stick to Figma design, while being creative is crucial, in your workplace you may be requested to develop web pages as per the given design. Therefore I recommend you to try to match the original navbar design:

Your design  | Figma design
------------- | -------------
<img width="377" alt="Screenshot 2022-03-15 at 21 41 42" src="https://user-images.githubusercontent.com/77038610/158438803-f2ff56ce-240d-44d3-9e0f-0adba0e41b31.png"> | <img width="373" alt="Screenshot 2022-03-15 at 21 42 07" src="https://user-images.githubusercontent.com/77038610/158438883-ded1df77-37ee-4736-8ae6-4fa09e51bcf7.png">

More specifically: 
Please decrease the `my logo` size and increase the hamburger icon size. Cange the icon color from `blue` to `#6070FF`